src/components/screens/lockedBalance/lockedBalance/form.js

```diff
@@ -53,12 +53,16 @@ const Form = ({
       nonce: account.nonce,
       fee: `${toRawLsk(parseFloat(selectedFee))}`,
       passphrase: account.passphrase,
-      unlockingObjects: getAvailableUnlockingTransactions(account, currentBlock),
+      unlockingObjects: getUnlockableUnlockingObjects(account, currentBlock),
```

Comment:
I think `getUnlockableUnlockingObjects` is a bit confusing name. In this context `getUnlockingObjects` explains itself better

Reply:
the function filters the unlocking objects and only returns the `unlockingObjects` that are `unlockable` so `getUnlockable-UnlockingObjects` sounds like a very fitting name to me 😃